doc/internal/man3/ossl_param_bld_init.pod

```diff
@@ -90,6 +92,14 @@ will also be securely allocated.
 The I<bn> argument is stored by reference and the underlying BIGNUM object
 must exist until after ossl_param_bld_to_param() has been called.
 
+ossl_param_bld_push_BN_pad() is a function that will create an OSSL_PARAM object
+that holds the specified BIGNUM I<bn>.
+The object will be padded to occupy exactly I<sz> bytes.
```

Comment:
What happens when *sz* has a smaller value than the size of the bignum?  What happens when *sz* is zero?

Reply:
Added a note indicating an error will occur when insufficient space is specified.